CMakeLists.txt

```diff
@@ -69,6 +73,10 @@ else()
 	add_definitions(-Wno-reorder -pedantic-errors -Ofast -fno-fast-math)
 endif()
 
+if(MAC_OS)

```

Comment:
For now this is okay, but in the future we want to move away from piling onto a sourcelist.
Perhaps you could add a CMake target instead?

Reply:
I am not familiar with CMake. Will investigate how to do that.

Reply:
By the way, can you recommend some CMake tutorial?

Reply:
Checkout CppCon talks on CMake.
I can also recommend this: https://cliutils.gitlab.io/modern-cmake/

Reply:
Thanks a lot